tests/integration/blockchain/test_exact_economics_model.py

```diff
@@ -83,21 +85,23 @@ def test_exact_economics():
         # Sanity check expected testing outputs
         assert Decimal(expected_total_supply) / expected_initial_supply == expected_supply_ratio
         assert expected_reward_supply == expected_total_supply - expected_initial_supply
-        assert reward_saturation * 365 * multiplier == expected_lock_duration_coefficient_1 * (1 - multiplier)
-        assert int(365 ** 2 * reward_saturation * decay_half_life / log(2) / (1-multiplier) / expected_lock_duration_coefficient_2) == \
+        assert reward_saturation * one_year_in_periods * multiplier == expected_lock_duration_coefficient_1 * (1 - multiplier)
+        assert int(one_year_in_periods ** 2 * reward_saturation * decay_half_life / log(2) / (1-multiplier) / expected_lock_duration_coefficient_2) == \
             expected_phase2_coefficient
 
+
+
     # After sanity checking, assemble expected test deployment parameters
     expected_deployment_parameters = (24,       # Hours in single period at genesis
-                                      24,       # Hours in single period
-                                      1053,     # Coefficient which modifies the rate at which the maximum issuance decays (d)
-                                      365,      # Numerator of the locking duration coefficient (k1)
-                                      730,      # Denominator of the locking duration coefficient (k2)
-                                      365,      # Max periods that will be additionally rewarded (awarded_periods)
-                                      2829579800000000000000000000,  # Total supply for the first phase
-                                      1002509479452054794520547,     # Max possible reward for one period for all stakers in the first phase
-                                      30,       # Min amount of periods during which tokens can be locked
-                                      15000000000000000000000,       # min locked NuNits
+                                      24 * 7,   # Hours in single period
+                                      150,     # Coefficient which modifies the rate at which the maximum issuance decays (d)
+                                      52,      # Numerator of the locking duration coefficient (k1)
+                                      52 * 2,  # Denominator of the locking duration coefficient (k2)
+                                      52,      # Max periods that will be additionally rewarded (awarded_periods)
+                                      2829579800000000000000000000,   # Total supply for the first phase
+                                      7017566356164383151812537,      # Max possible reward for one period for all stakers in the first phase
```

Comment:
Hazy on the calculation of this value...? I assumed `1002509479452054794520547 x 7` but not yielding the same result.

```python
In [37]: 1002509479452054794520547 * 7
Out[37]: 7017566356164383561643829
```

Reply:
we get all values from perspective that year is ~52.14 weeks, but in the result we can have only integer numbers. We have 5 years for first phase (at least), whic still not integer value in periods, that's why the difference. (See StandardEconomics init)